src/pages/HomePage/ListOfHouses/ListOfHouses.module.scss

```diff
@@ -14,29 +14,30 @@
   overflow-y: scroll;
 }
 
-.housesList::-webkit-scrollbar {
-  width: 10px;
-}
+// .housesList::-webkit-scrollbar {
+//   width: 10px;
+// }
 
-.housesList::-webkit-scrollbar-track {
-  background: #f1f1f1;
-  border-radius: 10px;
-}
+// .housesList::-webkit-scrollbar-track {
+//   background: #f1f1f1;
+//   border-radius: 10px;
+// }
 
-.housesList::-webkit-scrollbar-thumb {
-  background: #888;
-  border-radius: 10px;
-}
+// .housesList::-webkit-scrollbar-thumb {
+//   background: #888;
+//   border-radius: 10px;
+// }
 
-.housesList::-webkit-scrollbar-thumb:hover {
-  background: #555;
-}
+// .housesList::-webkit-scrollbar-thumb:hover {
+//   background: #555;
+// }
 
 .houseElement {
```

Comment:
Something's not right with house elements if no photos are displayed.
![Screenshot from 2022-02-04 20-27-58](https://user-images.githubusercontent.com/47178494/152591506-316313f4-3b29-4597-b50e-c8704b5521d0.png)